src/network/clientpackethandler.cpp

```diff
@@ -120,7 +121,12 @@ void Client::handleCommand_AuthAccept(NetworkPacket* pkt)
 					<< m_recommended_send_interval<<std::endl;
 
 	// Reply to server
-	NetworkPacket resp_pkt(TOSERVER_INIT2, 0);
+	std::string lang = gettext("LANG_CODE");
```

Comment:
I think it's fine to add it here, but DO NOT use << with a plain string. It puts the raw string there with no length information, which makes version compatibility harder if other stuff is added after it. IMO it needs a length-prefixed string, which is what is usually used in the protocol. For example `resp_pkt << serializeString(lang);`.